slim/dzslides/document.html.slim

```diff
@@ -24,6 +24,13 @@ html lang=(attr :lang, 'en' unless attr? :nolang) class="aspect-#{attr 'dzslides
         style
           =endline
           =::IO.read %(./dzslides/themes/highlight/#{attr 'dzslides-highlight', 'default'}.css)
+    - if attr? 'stylesheet'
+          - if link_assets
+            link rel='stylesheet' href=normalize_web_path("#{attr 'stylesheet'}", (attr :stylesdir, ''))
+          - else
+            style
+              =endline
+              =::IO.read %(#{attr 'stylesdir','.'}/#{attr 'stylesheet'})
```

Comment:
Could you write this line as follows:

```ruby
=read_asset(normalize_system_path((attr 'stylesheet'), (attr :stylesdir, '')), :warn_on_failure => true)
```

The `normalize_system_path` handles all the concerns of locating the file to read. The `read_asset` method reads and normalizes the file so that we don't get any encoding problems. Thanks!